rfc6962-bis.xml

```diff
@@ -1208,21 +1209,17 @@ the STH changes.
           </list>
         </t>
       </section>
-      <section title="Auditor">
+      <section title="Auditing">
         <t>
-          Auditors take partial information about a log as input and verify
-that this information is consistent with other partial information they
-have. An auditor might be an integral component of a TLS client; it might be a
-standalone service; or it might be a secondary function of a monitor.
+          Auditing is taking partial information about a log as input and verifying 
+          that this information is consistent with other partial information held. 
+          All clients described above may perform auditing as an additional function.
         </t>
         <t>
-          Any pair of STHs from the same log can be verified by requesting a consistency proof (<xref target="fetch_consistency"/>).
+          A <xref target="Monitor">monitor</xref> can audit by verifying the consistency of STHs it receives and verifying that each entry fetched is included in the tree.
```

Comment:
And that it can fetch every entry in the tree, and the that STH is indeed the result of making a tree from all fetched entries.